bc_obps/reporting/service/report_submission_service.py

```diff
@@ -50,11 +51,12 @@ def submit_report(version_id: int, user_guid: UUID, sign_off_data: ReportSignOff
         ReportSubmissionService.validate_report(version_id)
         ReportSignOffService.save_report_sign_off(version_id, sign_off_data)
         # Mark the previous latest submitted version as not latest
-        ReportVersion.objects.filter(
-            report=report_version.report,
-            status=ReportVersion.ReportVersionStatus.Submitted,
-            is_latest_submitted=True,
-        ).update(is_latest_submitted=False)
+        with pgtrigger.ignore("reporting.ReportVersion:set_updated_audit_columns"):
+            ReportVersion.objects.filter(
+                report=report_version.report,
+                status=ReportVersion.ReportVersionStatus.Submitted,
+                is_latest_submitted=True,
+            ).update(is_latest_submitted=False)
 
         # Set the new version as lastest submitted
         report_version.is_latest_submitted = True
```

Comment:
we also need to wrap the `submit_report()` method in a `@transaction.atomic`, otherwise we have a short time where the database is in an inconsistent state (no latest submitted version)